eng/config/globalconfigs/Common.globalconfig

```diff
@@ -71,8 +71,8 @@ dotnet_diagnostic.RS0006.severity = error
 dotnet_diagnostic.RS0012.severity = warning
 dotnet_diagnostic.RS0014.severity = warning
 dotnet_diagnostic.RS0015.severity = warning
-dotnet_diagnostic.RS0016.severity = error
-dotnet_diagnostic.RS0017.severity = error
+dotnet_diagnostic.RS0016.severity = none # PROTOTYPE(ndsm): re-enable
```

Comment:
Should the PROTOTYPE comments in this PR be addressed?

Reply:
That's from @RikkiGibson 's commit. I would have to refer back to him to see how this needs to be updated.

Reply:
First step I think is to present the findings to API review. They may want revisions to names, shapes, etc. of the API but any changes should have pretty small impact on this PR.

My commits haven't yet been reviewed by compiler team so we will need 2 compiler team reviews on the PR as well. I would be fine with just pushing commits to this PR to address any feedback.

This specific prototype comment will be addressed by re-enabling the error and updating the public API files.